### PR TITLE
Mogh99 development

### PR DESCRIPTION
**Pull request by mogh99**
Hi,
This pull request provides some changes to the qubes menu. Adding light, and dark mode. Fixing some bugs and changing the layout as shown below.

fav_page_dark

fav_page_light

loading_vm

qube_page

### CODE REVIEW

**Comment by marmarek** [comment]
PipelineRefresh

File: Makefile [hunk 1]
```diff
@@ -19,14 +21,37 @@ install-icons:
...
+	cp icons/qappmenu-list-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-list-black.svg
+	cp icons/qappmenu-grid-white.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-grid-white.svg
+	cp icons/qappmenu-grid-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-grid-black.svg
+	cp icons/qappmenu-sun-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-sun-black.svg
+	cp icons/qappmenu-sun-white.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-sun-white.svg
+	cp icons/qappmenu-moon-black.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-moon-black.svg
+	cp icons/qappmenu-services.svg $(DESTDIR)/usr/share/icons/hicolor/scalable/apps/qappmenu-services.svg
+
+update-icons:
+	gtk-update-icon-cache $(DESTDIR)/usr/share/icons/hicolor
```
Comment (marmarek) [inline Makefile:38]: This should be called at install time, not build time.
See how it's done in desktop-linux-manager, especially https://github.com/QubesOS/qubes-desktop-linux-manager/blob/master/rpm_spec/qubes-desktop-linux-manager.spec.in#L90 (and the `touch` call in %post)

**Comment by mogh99** [comment]
I added the new icons and new files (qappmenu-settings.ini, and qubes-menu-light.css) to the rpm_spec (`qubes-desktop-linux-menu.spec.in`). Regarding the makefile I use these scripts to install new icons or update the files during development. I tried to figure out how to build the rpm but I couldn't.

**Comment by marmarek** [comment]
> I added the new icons and new files (qappmenu-settings.ini, and qubes-menu-light.css) to the rpm_spec (`qubes-desktop-linux-menu.spec.in`). 

I see you added a file in user home dir (`~/.config/...`) - package can't do that. Application should have a reasonable defaults built in, and config file can be used to override them _by the user_.

> Regarding the makefile I use these scripts to install new icons or update the files during development. I tried to figure out how to build the rpm but I couldn't.

It's okay to keep those targets in the makefile, but you can't call them during package build. Drop them from the `install` target (and for development simply call `make install update-icons install-settings` to include those).

**Comment by marmarek** [comment]
The file location itself is fine. You just can't include the default one (stored in home dir) in the package. The app should load some reasonable build-in defaults if the file is not there. And can also create the file with those defaults to make it easier for the user to find it.

Ideally, switching between light and dark mode should be possible within the menu itself (maybe that's already the case, I haven't checked), same for other settings. But it's fine to leave it as just config option for now.

**Comment by mogh99** [comment]
@marmarta @marmarek 
Any updates regarding the pull requests?
Or do I need to fix something to accept it?

**Comment by marmarta** [comment]
I'll review in this week! The last two weeks were super busy because of https://qubesos.3mdeb.com/

**Comment by marmarta** [comment]
So, for the start: I'm not a fan of the redesign, chiefly because I don't understand your reasoning behind it :) The old layout was research-based, so we knew that at least a significant percentage of research participants liked it and found it useful. I don't like the idea of going back to "the person who made it likes it so let's use it". I know Qubes has no UI guidelines (yet, this is on my to-do list...), but I'd like more justifications for layout/UI/UX changes.

**Comment by marmarta** [comment]
And a general comment towards light-dark mode; for consistency, I'd prefer to use the system settings (you can check if, when loading, background is lighter or darker than text) and not a per-application switch.

**Pull request closed without merging**